constraints-dev.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/inveniosoftware/invenio-records.git@feature/custom-fields
```

Comment:
Maybe we add a comment that this file is used only when a PR is targeting the `feature/branch`